internal/controller/nnfsystemstorage_controller.go

```diff
@@ -0,0 +1,632 @@
+/*
```

Comment:
The other controllers have an underscore after nnf in the filename.

I'm guilty of this too, since I've added datamovement/container profiles and didn't use an underscore, but it's bothered me.

Reply:
It bothers me too, but I wonder if we should switch them all to use no underscore? That's the name of the file as generated by kubebuilder. I think kubebuilder must have changed at some point over the years.

I'm open to suggestions.

Reply:
Ah, yes. I think we should change them all to be consistent and we should go with what Kubebuilder does. Probably outside the scope of this PR though, but if you're feeling motivated :wink: